content/en/synthetics/cicd_integrations/jenkins.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Jenkins
+kind: documentation
+description: 
```

Comment:
@peculiarism please fill this out!

Reply:
`Configure your Jenkins instance to run Synthetic tests in your CI/CD pipelines`